testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcSamlForceAuthnBrokerTest.java

```diff
@@ -0,0 +1,104 @@
+package org.keycloak.testsuite.broker;
```

Comment:
Please add the license comment copying from another file and changing to current year.